CHANGELOG.md

```diff
@@ -1,8 +1,19 @@
 # CHANGELOG
 
-## v0.1.1 [28 October 2024]
+## Unreleased [TBD]
 
-- Minor updates to examples for NAWEA workshop
+- Replaces the git ProFAST installation with a PyPI installation.
+- Removed dependence on external electrolyzer repo
+- Updated CI to use conda environments with reproducible environment artifacts
+
+## v0.1.2 [28 October 2024]
+
+- Minor updates to examples for NAWEA workshop.
+- Adds `environment.yml` for easy environment creation and GreenHEART installation.
+
+## v0.1.1 [22 October 2024]
+
+- ?
```

Comment:
It looks to me that we skipped `v0.1.2`? Should the comments under it go under `v0.1.1` so we can replace the question mark?

Reply:
I think this is from accidentally merging develop, which I'm trying to figure out how to fully undo.